tlslite/tlsrecordlayer.py

```diff
@@ -912,3 +928,45 @@ def _changeWriteState(self):
 
     def _changeReadState(self):
         self._recordLayer.changeReadState()
+
+
+    def writeHeartbeat(self, type, payload, padding_length):
```

Comment:
I meant renaming it, but good point that this will always send a request so the type is redundant.